### PR TITLE
[684] Upgrade the HAPI FHIR server to R4 (4.0.1)

### PR DESCRIPTION
**Pull request by plarocque4**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -34,43 +37,47 @@ gretty {
...
-    testCompile "org.eclipse.jetty:jetty-servlet:9.4.12.v20180830"
-    testCompile "org.eclipse.jetty.websocket:websocket-server:9.4.12.v20180830"
-    testCompile "org.eclipse.jetty:jetty-server:9.4.12.v20180830"
-    testCompile "org.eclipse.jetty:jetty-util:9.4.12.v20180830"
-    testCompile "org.eclipse.jetty:jetty-webapp:9.4.12.v20180830"
-    compile "com.helger:ph-schematron:5.0.6"
-    compile "javax.interceptor:javax.interceptor-api:3.1"
-
-    testCompile group: 'junit', name: 'junit', version: '4.12'
+    compile 'javax.interceptor:javax.interceptor-api:1.2'
```
Comment (hkong) [inline build.gradle:80]: just want to make sure: going from 3.1 (line 71) to 1.2 ?

Comment (plarocque4) [reply build.gradle]: Apparently 3.1 was a mistake and the latest version is 1.2.2. I have now updated to that version.

Comment (plarocque4) [reply build.gradle]: A mistake in the javax interceptor repo that is. It should never have gone in as 3.1.

**Comment by lpdecelles** [comment]
Looks good. I tested it with HOT and PAP

**Pull request merged**